addon/components/polaris-avatar.js

```diff
@@ -256,50 +221,58 @@ export default Component.extend({
       ? 0
       : nameString.charCodeAt(0) % avatarImages.length;
     return `${this.get('avatarSourcePath')}/avatar-${++avatarIndex}.svg`;
-  }).readOnly(),
+  }
 
   /**
    * Flag controlling whether the avatar initials should be rendered
-   * @property shouldShowInitials
-   * @private
    * @type {Boolean}
+   * @private
    */
-  shouldShowInitials: computed('initials', 'hasImage', function() {
-    let { initials, hasImage } = this.getProperties('initials', 'hasImage');
+  @computed('initials', 'hasImage')
+  get shouldShowInitials() {
+    let { initials, hasImage } = this;
     return initials && !hasImage;
-  }).readOnly(),
+  }
 
   /**
    * Flag controlling whether the avatar image should be rendered
-   * @property shouldShowImage
-   * @private
    * @type {Boolean}
+   * @private
    */
-  shouldShowImage: computed('finalSource', 'hasError', function() {
-    let { finalSource, hasError } = this.getProperties(
-      'finalSource',
-      'hasError'
-    );
+  @computed('finalSource', 'hasError')
+  get shouldShowImage() {
+    let { finalSource, hasError } = this;
     return finalSource && !hasError;
-  }).readOnly(),
-
-  didReceiveAttrs() {
```

Comment:
Why did this disappear?

Reply:
it's replaced by `resetImage` action below